aspnetcore/host-and-deploy/health-checks/samples/3.x/HealthChecksSample/BasicStartup.cs

```diff
@@ -0,0 +1,43 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace SampleApp
+{
+    // Use the `--scenario basic` switch to run this version of the sample.
+    //
+    // Register Health Check Middleware at the URL: /health
+    // 
+    // By default, health checks return a 200-Ok with 'Healthy'.
+    // - No health checks are registered by default. The app is healthy if it's reachable.
+    // - The default response writer writes the HealthStatus as text/plain content.
+    //
+    // This is the simplest way to perform health checks. It's suitable for systems that want to check for 'liveness' of an app.
+
+    #region snippet1
+    public class BasicStartup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddHealthChecks();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapHealthChecks("/health");
+            });
+
+            app.Run(async (context) =>
```

Comment:
Can the string `/` be a wildcard template? If it only responds to requests for root, I don't like it for this sample app. I'd like *every non-health check endpoint* to get the content.

Reply:
Also ...... this makes for an ugly sample app that displays code in the topic. I having to show this code now because it appears inside of the `UseEndpoints` snippets.

I don't think we should do this this way. Let's just place a note in the sample app that shows "the right way" and leave it out so that these don't show up in the topic.

Reply:
Naaaaaaaah. Rubber :duck: says to go ahead and inline the code examples in the topic.

Ok ... I'll fix it all up on the next commit.